cmd/testing-server/main.go

```diff
@@ -27,15 +28,16 @@ func runServer() error {
 	log.SetFlags(0)
 
 	router := mux.NewRouter()
-	router.HandleFunc("/create/{device-name}", createDeviceHandler)
 	router.HandleFunc("/", homeHandler)
+	router.HandleFunc("/create/{device-name}", createDeviceHandler)
 	router.HandleFunc("/triggerEvent/{type}", triggerEventHandler)
-	router.HandleFunc("/sendCPTVFrames", sendCPTVFramesHandler)
+	router.HandleFunc("/playCPTVFile", sendCPTVFramesHandler)
```

Comment:
I think this is mroe exat as sendCTVFrames or sendCPTVFrames ( i think this already implies a CPTV file), rather than play. i'ts not playing anything